app/controllers/notification_logs_controller.rb

```diff
@@ -17,7 +17,9 @@ def notification_log_params
       :document_type,
       :emailing_app,
       :publishing_app,
-      gov_delivery_ids: []
+      gov_delivery_ids: [],
```

Comment:
We currently plan to store all three gov_delivery_id, enabled_gov_delivery_id and disabled_gov_delivery_id as it gives us the most flexibility for reporting and has minimal costs.

Happy to discuss if you think it is worth removing this field in the future.